libraries/apollo-mockserver/src/commonMain/kotlin/com/apollographql/apollo3/mockserver/MockServer.kt

```diff
@@ -33,11 +51,50 @@ interface MockServerInterface {
   fun takeRequest(): MockRequest
 }
 
+expect fun MockServer(mockServerHandler: MockServerHandler = QueueMockServerHandler()): MockServer
+
+@Deprecated("Use enqueueString instead", ReplaceWith("enqueueString"), DeprecationLevel.ERROR)
+@ApolloDeprecatedSince(ApolloDeprecatedSince.Version.v4_0_0)
+fun MockServer.enqueue(string: String = "", delayMs: Long = 0, statusCode: Int = 200) = enqueueString(string, delayMs, statusCode)
+
+fun MockServer.enqueueString(string: String = "", delayMs: Long = 0, statusCode: Int = 200) {
+  enqueue(MockResponse.Builder()
+      .statusCode(statusCode)
+      .body(string)
+      .delayMillis(delayMs)
+      .build())
+}
+
+@ApolloExperimental
+interface MultipartBody {
+  fun enqueuePart(bytes: ByteString, isLast: Boolean)
+  fun enqueueDelay(delayMillis: Long)
+}
```

Comment:
This is the new, breaking multipart API.

Instead of `MockServer.enqueueMultipart(List<String>, ...)`, there is now a `MockServer.enqueueMultipart(): MultipartBody` which allows to:

1. enqueue binary parts if we want to
2. handle delays in a more central place
3. hide `ChunkedResponse`: multipart responses are always chunked with this API. If no chunk encoding is required, another body builder can be used.